inc/template-tags.php

```diff
@@ -231,9 +233,7 @@ function visualcomposerstarter_comment( $comment, $args, $depth ) {
 		<div class="comment-wrapper">
 			<footer class="comment-meta">
 				<div class="comment-author vcard">
-					<?php
-					/* translators: 1: author link */
-					printf( esc_html__( '%s</cite> <span class="says">says:</span>' ), get_comment_author_link() ); ?>
+					<?php echo sprintf( '%s %s %s %s'.esc_html__("says:", "visual-composer-starter").'%s ', '<cite>', get_comment_author_link(), '</cite>', '<span class="says">', '</span>' ); ?>
```

Comment:
This will be a better solution 
`<?php  printf( esc_html__("%s %s %s says: %s", "visual-composer-starter"), '<cite>', get_comment_author_link(), '</cite>', '<span class="says">', '</span>' ); ?>`